spiketools/utils/check_spikes_in_ms.py

```diff
@@ -0,0 +1,24 @@
+"""Spike related checking functions."""
+
+import numpy as np
+import warnings
+
+###################################################################################################
+###################################################################################################
+
+def check_spikes_in_ms(spikes):
```

Comment:
I think we could perhaps update this function name to `check_spike_time_unit`. 
The reason is I think this name is a little more "future proof" - in that in the future we might add / extend this function to be slightly more general in terms of checking spike time units.